vpr/src/route/rr_graph_indexed_data.cpp

```diff
@@ -451,7 +451,7 @@ static void load_rr_indexed_data_T_values() {
 static void calculate_average_switch(int inode, double& avg_switch_R, double& avg_switch_T, double& avg_switch_Cinternal, int& num_switches, short& buffered, vtr::vector<RRNodeId, std::vector<RREdgeId>>& fan_in_list) {
     auto& device_ctx = g_vpr_ctx.device();
     const auto& rr_graph = device_ctx.rr_graph;
-    const auto& rr_nodes = device_ctx.rr_nodes.view();
+    const auto& rr_nodes = rr_graph.rr_nodes().view();
```

Comment:
I do not think we still need a view here. Can you try to delete this line? For LINE 466, you can use the rr_graph.
If it does work, we keep this line.